src/components/item/header/ItemHeaderActions.js

```diff
@@ -87,6 +88,7 @@ const ItemHeaderActions = ({ onClickMetadata, onClickChatbox, item }) => {
             </Tooltip>
           )}
           <ShareButton itemId={id} />
+          <PublishButton itemId={id} />
```

Comment:
Most probably we want to display this button only when the item is public and the user is an admin.